standards-catalogue/data/organisations.yml

```
@@ -4,3 +4,18 @@ threesixtygiving:
 ietf:
   name: IETF
   url: 'https://www.ietf.org/'
+odf:
+  name: OASIS ODF Technical Committee
+  url: 'https://www.oasis-open.org/'
+openreferraluk:
```

For better readability, can we please make these kebab-case?
```suggestion
open-referral-uk:
```